pkgs/by-name/mu/mutmut/package.nix

```diff
@@ -1,39 +1,47 @@
-{ lib
-, fetchFromGitHub
-, python3
-, testers
+{
+  lib,
+  fetchFromGitHub,
+  mutmut,
+  python3Packages,
+  testers,
 }:
 
-let self = with python3.pkgs; buildPythonApplication rec {
+python3Packages.buildPythonApplication rec {
   pname = "mutmut";
-  version = "2.2.0";
+  version = "3.2.0";
 
   src = fetchFromGitHub {
     repo = pname;
     owner = "boxed";
-    rev = version;
-    hash = "sha256-G+OL/9km2iUeZ1QCpU73CIWVWMexcs3r9RdCnAsESnY=";
+    rev = "refs/tags/${version}";
+    hash = "sha256-+e2FmfpGtK401IW8LNqeHk0v8Hh5rF3LbZJkSOJ3yPY=";
   };
 
   postPatch = ''
-    substituteInPlace requirements.txt --replace 'junit-xml==1.8' 'junit-xml==1.9'
+    substituteInPlace requirements.txt --replace-fail 'junit-xml==1.8' 'junit-xml==1.9'
   '';
 
-  disabled = pythonOlder "3.7";
+  disabled = python3Packages.pythonOlder "3.7";
 
   doCheck = false;
 
-  propagatedBuildInputs = [ click glob2 parso pony junit-xml ];
-
-  passthru.tests.version = testers.testVersion { package = self; };
```

Comment:
You'll want to retain this and do `package = mutmut``

Reply:
Oh okay, I thought you removed it only because you don't do the self definition anymore